lib/config.js

```diff
@@ -83,7 +83,7 @@ const applyOverrides = (config, options) => {
 
 const deepFreeze = object => {
   Object.keys(Object.freeze(object))
-    .map(key => object[key])
+    .map(key => object[key] && object[key] !== object.static)
```

Comment:
We don't want `config.npm.static` frozen just yet, because we need to be able to add `includes` from plugins to it after they are intialized